### PR TITLE
PXB-1794: Release notes for v8.0.5

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/conf.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
 # The short X.Y version.
 version = '8.0'
 # The full version, including alpha/beta/rc tags.
-release = '8.0.4'
+release = '8.0.5-1'
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/conf.py]: As far as I know we haven't change release numbering rules. The release number should be 8.0.5 and -1 is the part of the package build number.

**Review by gl-sergei — CHANGES_REQUESTED** [review]
Please also see updated 8.0.5 milestone for missing issues.

File: storage/innobase/xtrabackup/doc/source/conf.py (hunk 1 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/conf.py]: Ready for review

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.5.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- :jirabug:`1781`: Added support for binary log encryption introduced
+  in |MySQL| 8.0.14.
+
+Bugs Fixed
+================================================================================
+
+- Using ``--lock-ddl-per-table`` caused the server to scan all records of
+  partitioned tables which could lead to the "out of memory" error. Bugs fixed
+  :jirabug:`1691` and :jirabug:`1698`.
+- Incremental backups did not update ``xtrabackup_binlog_info`` with
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.5.rst]: 8.0 was not affected, I only merged the test case from 2.4 branch

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.5.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  ``--binlog-info=lockless``. Bug fixed :jirabug:`1711`.
+- When |pxb.name| was started run with the :option:`--slave-info`, incorrect
+  coordinates were written to the xtrabackup_slave_info file.. Bug fixed :jirabug:`1737`
+- |pxb.name| could crash at the prepare stage when making an
+  incremental backup if the variable ``innodb-rollback-segments`` was
+  changed after starting the |MySQL| Server. Bug fixed
+  :jirabug:`1785`.
+- The full backup could fail when |Percona Server| was started with encryption
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.5.rst]: specifically, when server was initialized with --innodb-encrypt-tables option

**Review by gl-sergei — CHANGES_REQUESTED** [review]
Just a couple of minor comments

**Review by gl-sergei — APPROVED** [review]
Looks good. Thank you!

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.5.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  changed after starting the |MySQL| Server. Bug fixed
+  :jirabug:`1785`.
+- The full backup could fail when |Percona Server| was started with the ``--innodb-encrypt-tables``
+  parameter. Bug fixed :jirabug:`1793`.
+
+Other bugs fixed: :jirabug:`1632`, :jirabug:`1715`, :jirabug:`1770`, :jirabug:`1771`, :jirabug:`1773`.
+
+.. |percona| replace:: Percona
+.. |pxb.name| replace:: Percona XtraBackup
+.. |date| replace:: February 27, 2018
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.5.rst]: please change release date to March 04, *2019*

**Pull request merged**